modules/ROOT/pages/monitoring-dashboards.adoc

```diff
@@ -160,24 +164,24 @@ Threads here refer to threads of execution in Java. The JVM allows an applicatio
 [[garbage_collection]]
 === Garbage Collection
 
-The JVM implements the garbage collection mark-and-sweep algorithm, which consists of two phases:
+Garbage collection involves examining the heap memory to identify which objects are not used, and then deleting them. The garbage collection graph shows metrics for these areas of the heap:
 
-* Phase One: This is the mark phase, where the garbage collection algorithm finds and marks all accessible objects. 
-* Phase Two: This is the sweep phase, where the garbage collection algorithm scans through the heap and reclaims all the unmarked objects. 
+* Young Generation where all new objects are allocated.
+* Old Generation where long surviving objects are stored.
 
 image::monitoring-dashboards-garbage.png[]
 
 === Class Loading
 
-The class loading graph shows the amount of classes that are currently loaded, or being loaded, in total across all apps in the JVM.
+The class loading graph shows the number of classes that are currently loaded or being loaded, in total across all apps in the JVM.
 
 image::monitoring-dashboards-classloading.png[]
 
 === Heap Memory Metrics
```

Comment:
Why is this separate from Heap Memory?
Hmm...maybe these titles need modifiers, like Graph or Tab?